### DISCUSSION TITLE
What is M on the FDS Users Guide for running FDS on Linux? export OMP_NUM_THREADS=M

### DISCUSSION THREAD

**Discussion by Yunlongjasonliu** [firemodels/fds#13620] [General]
On page 13 of the FDS Users Guide, section 3.2.3 Linux and macOS, it reads:

export OMP_NUM_THREADS=M 
mpiexec -n N -hostfile hosts.txt /home/username/.../fds job_name.fds >& job_name.err &

Do I need to give a value for M, or I just leave the letter M there?
If I need to give a value for M, what number it should be?

Thanks

**Comment by drjfloyd** [comment]
M is a number that you choose.  Read the remainder of that chapter of the Guide for more information on OpenMP and MPI.  You may also want to go to the Wiki page whose link is in that section of the Guide.

**Comment by mcgratta** [comment]
Set M to 1 if your input file has multiple `MESH` lines. Typically you would then set N to the number of meshes.
